libs/device-core/src/commands/entities/AppStorageInfo.ts

```diff
@@ -5,3 +5,20 @@ export type AppStorageInfo = {
   hasData: boolean;
   hash: string;
 };
+
+export function appStorageInfoGuard(data: AppStorageInfo | unknown): data is AppStorageInfo {
```

Comment:
[ASK] IDK about the team sentiment but I would prefer `isAppStorageInfo` to explicitly notite that the function return a boolean.

Reply:
agreed

Reply:
Updated.